src/components/notifications/notificationBar.css

```
@@ -0,0 +1,64 @@
+.notification-bar-container {
+  width: 100%;
+  height: 64px;
+  margin-top: 12px;
+  margin-bottom: 12px;
+  position: relative;
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 0px;
+  gap: 15px;
+
+  border-left: 2px solid #f2f2f7;
```

Only have a blank line between selectors.

```suggestion
  position: relative;
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: center;
  padding: 0px;
  gap: 15px;
  border-left: 2px solid #f2f2f7;
```